patches/react-native-keyboard-controller+1.10.4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/react-native-keyboard-controller/src/components/KeyboardAvoidingView/index.tsx b/node_modules/react-native-keyboard-controller/src/components/KeyboardAvoidingView/index.tsx
+index 9e73caf..0d5f143 100644
+--- a/node_modules/react-native-keyboard-controller/src/components/KeyboardAvoidingView/index.tsx
++++ b/node_modules/react-native-keyboard-controller/src/components/KeyboardAvoidingView/index.tsx
+@@ -81,7 +81,7 @@ const KeyboardAvoidingView = forwardRef<View, React.PropsWithChildren<Props>>(
+     const onLayoutWorklet = useCallback((layout: LayoutRectangle) => {
+       "worklet";
+ 
+-      if (keyboard.isClosed.value) {
++      if (keyboard.isClosed.value || initialFrame.value === null) {
```

Comment:
@kirillzyusko isn't this a library you maintain? can we add this in the upstream?

Reply:
@roryabraham yes, I did quick patch-package just to verify that it actually fixes the problem. It fixes, so I pushed this changes to upstream and published a new version to `npm` and update `package.json` 👀